.github/workflows/test.yml

```diff
@@ -15,13 +15,14 @@ jobs:
       fail-fast: false
       max-parallel: 5
       matrix:
-        python-version: [ '3.8', '3.9', '3.10', '3.11']
-        django-version: ['3.2', '4.1', '4.2']
-        drf-version: ['3.12', '3.13', '3.14']
+        python-version: ['3.9', '3.10', '3.11', '3.12', '3.13']
```

Comment:
@Andrew-Chen-Wang , according to Django docs, [Django 4.2 still supports Python 3.8](https://docs.djangoproject.com/en/5.1/faq/install/#what-python-version-can-i-use-with-django).

I'm trying to parse the language in that section, but finding it a little confusing:

> Typically, we will support a Python version up to and including the first Django LTS release whose security support ends after security support for that version of Python ends. For example, Python 3.9 security support ends in October 2025 and Django 4.2 LTS security support ends in April 2026. Therefore Django 4.2 is the last version to support Python 3.9.

Based on ^, I think that it's correct that Django 4.2 still supports Python 3.8.

Apologies if I've missed discussion of this elsewhere.

Reply:
+1 to this, i want to believe there might have been some mistake leading to confusion somewhere, specifically around here:

> ...Django 4.2 LTS security support ends in April 2026. Therefore Django 4.2 is the last version to support Python 3.9.

data shows 3.8 is still supported by the framework versioned at 4.2, this statement here however, might just have lost that point - unless it would mean, `...4.2 is the last version to support both Python 3.8 and 3.9` phasing out both versions, perhaps?

Reply:
> unless it would mean, `...4.2 is the last version to support both Python 3.8 and 3.9` phasing out both versions, perhaps?

This is also how I interpret it, based on the described method of determining support.

Reply:
Python 3.8 has reached EOL for its security status, so I have dropped it.